projects/storefrontapp-e2e-cypress/cypress/helpers/cart.ts

```diff
@@ -1,6 +1,6 @@
 import { standardUser } from '../sample-data/shared-users';
 import { apiUrl } from '../support/utils/login';
-import { login } from './auth-forms';
+import { login, register } from './auth-forms';
```

Comment:
Speed of code review is crucial (not how fast you do it, but how fast can you start reviewing).

Reply:
https://miro.medium.com/max/507/1*RANwdB5NtwjDTeFaPaVv_g.jpeg

Reply:
![1_RANwdB5NtwjDTeFaPaVv_g](https://user-images.githubusercontent.com/6259079/144898491-de494361-322c-429e-a48a-16390e40ff03.jpeg)